crds/jwst/tpns/nirspec_cubepar.tpn

```diff
@@ -1,6 +1,6 @@
 
 # Make EXP_TYPE available in headers even if only UNDEFINED
 # Keyword cross-strapping doesn't work for UNDEFINED values...
-META.EXPOSURE.TYPE         H   C   R   NRS_IFU
+META.EXPOSURE.TYPE         H   C   R   NRS_AUTOFLAT,NRS_AUTOWAVE,NRS_IFU,NRS_LAMP
```

Comment:
Would it be more future proof to just eliminate this line and accept any `NRS_` exposure type from the list in nirspec_all.tpn?  Or are some of those exposure types invalid?

Reply:
For the reference file that's being delivered, these are the only exposure types that are valid.